src/componentprocessor/index.ts

```diff
@@ -156,24 +156,6 @@ async function processXmlTree(
         if (xmlNode) {
             nodeDef.children = getChildren(xmlNode);
             nodeDef.scripts = getScripts(xmlNode);
-            let baseNode = xmlNode.attr.extends;
-            while (baseNode) {
-                let baseNodeDef = nodeDefMap.get(baseNode);
-                if (baseNodeDef) {
-                    nodeDef.children = [...nodeDef.children];
-                    baseNode = baseNodeDef.xmlNode!.attr.extends;
-                } else {
-                    if (!(baseNode in BrsComponentName)) {
-                        // The reference implementation doesn't allow extensions of unknown node subtypes, but
-                        // BRS hasn't implemented every node type in the reference implementation!  For now,
-                        // let's warn when we detect unknown subtypes.
```

Comment:
Would you mind copying this comment over to `ComponentScopeResolver.ts` with our `console.error`?

Reply:
You ask so much of me! 😉